core/src/replay_stage.rs

```diff
@@ -437,6 +441,10 @@ impl ReplayStage {
                 };
                 let in_vote_only_mode = bank_forks.read().unwrap().get_vote_only_mode_signal();
 
+                // TODO (LB): dont do * 2
```

Comment:
```
lazy_static! {
    // reduce the number of threads each pool is allowed to half the cpu core count, to avoid rayon
    // hogging cpu
    static ref MAX_RAYON_THREADS: usize =
            env::var("SOLANA_RAYON_THREADS").ok()
            .and_then(|num_threads| num_threads.parse().ok())
            .unwrap_or_else(|| num_cpus::get() as usize / 2)
            .max(1);
}

pub fn get_thread_count() -> usize {
    *MAX_RAYON_THREADS
}
```

Reply:
gonna use num_cpus crate directly instead